.merlin

```diff
@@ -1,2 +0,0 @@
-B _build/**
-FLG -w A-4-44-48
```

Comment:
`jbuilder` generates accurate `.merlin` files when building -- no need to commit them.

Reply:
@cfcs, please keep the discussion respectful.

If you are building for the first time, there are no compiled artifacts to begin with, so having `.merlin` around will not help you much.

If "something breaks" after the first compile, you will have the `.merlin` files left over from the last compile, so am not sure I understand what is the problem.

On the other hand, having the build system generate `.merlin` files means that they are always kept up-to-date, a big improvement over manual maintenance.

Reply:
Having `.merlin` around from the beginning will still include the external libraries, and your settings for flags/warnings, at least that's the experience I have in `emacs` when I clean, then open with/without a `.merlin` file. This is useful when you're just checking out the repository to check the implementation of something, which is a common thing for me at least, in the face of the extremely rigorous documentation provided with `notty`.

If "something breaks" after the first compile, it depends on "what breaks, and when" I guess?
At least in other projects that used `jbuilder` it's commonly happened to me, but that's kind of out of our control in this case and that fight someone should probably bring to the jbuilder upstream repo instead of complaining about it here. :)

I agree that it's nice when something can suggest new things to be in the `.merlin` file, I just don't understand why jbuilder a) has to remove it all the time, meaning it can't be checked into VCS b) sometimes fails to produce a replacement for the one it deletes.

Reply:
You can always ask jbuilder to build .merlin files explicitly - even if your project is in a broken state.

Reply:
Oh, that would be nice!